src/content/docs/en/develop-and-build.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Develop and build with Astro
+description: 'How to start working in a new project.'
```

Comment:
Not sure if this should be `in` or `on`?
```suggestion
description: 'How to start working on a new project.'
```

Reply:
I do think this could go either way. I was probably feeling "jump IN to your new project and start working" vibes at the time. Perhaps "on" is the safer bet!